package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "devDependencies": {
+    "prettier": "^3.2.5"
+  }
```

Comment:
Is there a way to have the markdown run without the tool metadata within the repo? Or at least to move it to a ".hidden" directory? It would be preferred not to take the additional ecosystem tooling if it's possible. Is there a containerized version of this?

Reply:
Good point. I switched over to use go formatters.